### PR TITLE
Improve consistency on city names and adding a validator

### PR DESCRIPTION
**Pull request by jespino**
I added a check for the geolocations to detect duplications, normally means that you have written the city with two different spellings, that can lead to hidding pins in the map, so for consistency I'm checking that. This PR also applies the changes in the names of the cities to keep them consistent.

### CODE REVIEW

File: page/src/misc/geolocations.json [hunk 1]
```diff
@@ -2487,18 +2243,13 @@
...
-    "latitude": 45.6306954,
-    "longitude": -122.6744557
-  },
   "Vancouver, BC (Canada)": {
     "latitude": 49.2608724,
     "longitude": -123.113952
   },
+  "Vancouver, BC (USA)": {
+    "latitude": 49.2827291,
+    "longitude": -123.1207375
```
Comment (scraly) [inline page/src/misc/geolocations.json:2252]:
```suggestion
    "longitude": -123.1207375
},
```

**Review by scraly — APPROVED** [review]

**Comment by scraly** [comment]
Thanks, it's a good idea :) 

The script found 2 duplicated coordinates:

```
Duplicate coordinates: Bangalore (India) and Bengaluru (India)
Duplicate coordinates: Bengaluru (India) and Bangalore (India)
Duplicate coordinates: Detroit (USA) and Detroit, MI (USA)
Duplicate coordinates: Detroit, MI (USA) and Detroit (USA)
```

Can you fix them in your branch please?

**Comment by jespino** [comment]
@scraly fixed, also I added a more diffuse way of checking the coincidence, from exact match to really close match. That way I was able to capture more duplicates.

**Comment by scraly** [comment]
It's better thanks :)

**Comment by scraly** [comment]
FYI some other duplicate cities exist:

```
  "Belo Horizonte (Brazil)": {
    "latitude": -19.919052,
    "longitude": -43.9386685
  },
  "Belo Horizonte, MG (Brazil)": {
    "latitude": -19.9227318,
    "longitude": -43.9450948
  },
 ```

If you want to improve your script :)

**Comment by scraly** [comment]
And:

```
  "Buenos Aires (Argentina)": {
    "latitude": -34.6075682,
    "longitude": -58.4370894
  },
  "Buenos Ares (Argentina)": {
    "latitude": -34.63974725,
    "longitude": -58.4446923529756
  }
 ```

**Pull request merged**